ingeniamc/k2-base-camp
# DR0053A-591 Select separate files

### Docs

https://novantamotion.atlassian.net/browse/DR0053A-591

The user should have the option to upload separate dictionaries for each drive.
This PR was used to also update the interface to select configurations for the drive.

### Test

1. Play around with the new interface, making sure that it behaves as you'd expect and allows you to simply and intuitively select the same or different configurations and / or dictionaries for the drives.
2. Connect to the drives using two different dictionaries and make sure that this works as expected.
3. To the same when selecting two different configurations.
4. Make sure that selecting the same configuration / dictionary for both drives using the combined button still works as it should.
